tests/cairo_files/instructions/test_exchange_operations.cairo

```diff
@@ -15,43 +15,62 @@ from kakarot.model import model
 from kakarot.stack import Stack
 from tests.utils.utils import TestHelpers
 
-// @notice Prepare a stack with `idx` elements to test swap logic. 
-func prep_stack{range_check_ptr}(idx : felt, to_swap_idx: felt,  stack : model.Stack*) -> (prepped_stack: model.Stack*) {
+// @notice Prepare a stack with `stack_len` elements to test swap logic. 
+func prepare_stack{range_check_ptr}(stack_len : felt, preswap_idx: felt, preswap_idx_value: Uint256, top_stack_element_value: Uint256,  stack : model.Stack*) -> (prepared_stack: model.Stack*) {
```

Comment:
`preswap_idx` -> `swap_idx`
`preswap_idx_value` -> `swap_idx_value`
`top_stack_element_value` -> `top_stack_value` to keep consistency